### DISCUSSION TITLE
There are no details for configuring DNS over TLS or HTTPS.

### DISCUSSION THREAD

**Discussion by dishoneprabu** [0xERR0R/blocky#1604] [Q&A]
Can anyone help me on this

![image](https://github.com/user-attachments/assets/9f5a8be0-864a-41a6-b2b7-8e6e923856f9)

**Comment by TheoTechnicguy** [comment]
Hey! Can you elaborate on what you expect or where you need help?

Most of the configuration is already present elsewhere in the documentation. As far as my experience goes, the resolver uses the x509 TLS/SSL certificates you provide in [basic configuration](https://0xerr0r.github.io/blocky/latest/configuration/#basic-configuration) and opens listeners on ports specified in the [ports configuration](https://0xerr0r.github.io/blocky/latest/configuration/#ports-configuration) section.

Basically, to get a server running, you simply have to give your server a x509 certificate bundle and write what ports to open in the configuration. Something like below.

```yaml
certFile: server.crt
keyFile: server.pem

ports:
  dns: 53
  tls: 853
  https: 443
```
